_field-types/metadata-fields/source.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Source
+nav_order: 40
+has_children: false
+parent: Metadata fields
+---
+
+# Source
+
+The `_source` field contains the original JSON document body that was passed at index time. This field is not indexed and is not searchable but is stored so that it can be returned when executing fethc requests such as `get` and `search`.
+
+## Disabling the field
+
+You can disable the `_source` field as shown in the following example:
+
+```json
+PUT sample-index1
+{
+  "mappings": {
+    "_source": {
+      "enabled": false
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+Disabling the `_source` field can cause certain featues to become not supported, such as `update`, `update_by_query`, and `reindex` APIs and the ability to debug queries or aggregations by using the the original document at index time.
+{: .warning}
+
+## Including or excluding fields
+
+You can prune contents of the `_source` field after the document has been indexed but before the field is stored. You can use the `includes` and excludes` parameters to control which field are included in the stored `_source` field, as shown in the following example:
```

Comment:
```suggestion
You can prune contents of the `_source` field after the document has been indexed but before the field is stored. You can use the `includes` and `excludes` parameters to control which field are included in the stored `_source` field, as shown in the following example:
```

(randomly stumbled over this 😅 - noticed it because of the syntax highlighting on GH)